improver/wxcode/modal_code.py

```diff
@@ -205,6 +215,17 @@ def process(self, cubes: CubeList) -> Cube:
             result = cube.collapsed("time", self.aggregator_instance)
         self._set_blended_times(result)
 
+        if self.model_id_attr:
+            # Update contributing models
+            contributing_models = set()
+            for source_cube in cubes:
+                for model in source_cube.attributes[self.model_id_attr].split(" "):
+                    contributing_models.update([model])
+            # iris concatenates string coordinates as a "|"-separated string
```

Comment:
Why this comment?

Reply:
I've no idea. It must have meant something to me once. I've removed it.